speechbrain/core.py

```diff
@@ -1236,6 +1259,36 @@ def evaluate(
         self.step = 0
         return avg_test_loss
 
+    @contextmanager
+    def no_sync(self, use=True):
```

Comment:
Is this affecting the training time? If so, it also looks like something that should be put on an other PR because it affects ALL the recipes and not just this one.